app/Providers/AppServiceProvider.php

```diff
@@ -51,12 +52,16 @@ static function () {
             }
         );
 
+
+
         Sendportal::setHeaderHtmlContentResolver(
             static function () {
                 return view('layouts.header.userManagementHeader')->render();
             }
         );
 
+        Paginator:: useBootstrap();
```

Comment:
Given that this has effects on the user's entire application, I don't believe that it's the responsibility of the package to include this.  That said, it belongs more in the documentation.